src/main/java/edu/byu/cs/autograder/PhaseThreeGrader.java

```diff
@@ -16,4 +16,11 @@ public class PhaseThreeGrader extends PassoffTestGrader {
     public PhaseThreeGrader(String netId, String repoUrl, Observer observer) throws IOException {
         super("./phases/phase3", netId, repoUrl, observer, Phase.Phase3);
     }
+
+    /* Rubric Items Winter 2024:
```

Comment:
Are you sure you meant to include these time-specific comments in the code? It would seem prudent to remove them if possible, or at least include the full details if it's necessary that they continue. (It looks like the beginning portion of some of the fields have been omitted.)

A better solution would be to include some documentation in the `README.md` file that shows where the actual rubrics should be managed. Since this is commented out, I'm assuming that the actual values are stored in a table somewhere. That should be described.

Reply:
There is no table that you assumed exists

Reply:
@19mdavenport Well then I'm even more confused. What is the purpose of including these rubric items in this comment?

Reply:
It was just a help for whoever is doing the phase 3 grader code so they don't have to look it up again